README.md

```diff
@@ -52,50 +43,38 @@ _NOTE:_
 - The measurement of TEE, Guest's boot, OS is per CC VM, but cluster/container measurement
 might be per cluster/namespace/container for cloud native architecture.
 - Please refer structure [`TDREPORT`](https://github.com/tianocore/edk2/blob/master/MdePkg/Include/IndustryStandard/Tdx.h)
+- CCNP project collects container level primitives by implementing unified APIs defined in [CC Trusted API](https://github.com/cc-api/cc-trusted-api). The project will be moved to [CC Trusted API](https://github.com/cc-api/cc-trusted-api) in the near future. 
 
 
-## 2. Design
-
-`CCNP` includes several micro-services as BaaS(Backend as a Service) to provides
-cloud native measurement, then exposes `CC trusted API` via cloud native SDK:
-
-- Services are designed to hide the complexity of different TEE platforms and provides
-common interfaces and scalability for cloud-native environment to address the fetching
-the fetching of quote, measurement and event log.
+## 2. Installation
 
-- SDK is provided to simplify the use of the service interface for development,
-it covers communication to the service and parses the results from the services.
-With such SDK, users can perform related actions with one simple API call.
+### 2.1 Configuration
```

Comment:
Could each of these sections apply 100% to either the host or the confidential VM?  If so I would put  "2.1 Configuration (HOST)"

"2.2 Deploy CCNP Service (Confidential VM"

"2.3 Install SDK"   I am not sure if this would be  confidential VM   or  container  I would try to write each section so that all the instructions are run 100% on a host  or 100% on a VM. It would help the user know where they should be installing things.   I don't know if this can be cleanly done when comparing confidential cluster in a VM vs confidential node as a VM  or confidential workload as a VM.  The key thing about my comment is it gets confusing for the user to know where to run the commands.